presto-main/src/main/java/com/facebook/presto/server/ServerInfoResource.java

```diff
@@ -92,4 +94,17 @@ public NodeState getServerState()
             return ACTIVE;
         }
     }
+
+    @GET
+    @Path("coordinator")
+    @Produces(TEXT_PLAIN)
+    public Response getServerCoordinator()
```

Comment:
With that other change, hitting / should return 200 on coordinator and 404 on workers. Though that might change in the future, so something more stable would be good.